openslides/users/signals.py

```diff
@@ -6,12 +6,13 @@
 
 def create_builtin_groups_and_admin(**kwargs):
     """
-    Creates the builtin groups: Anonymous, Registered, Delegates and Staff.
+    Creates the builtin groups: Anonymous, Registered, Delegates, Staff and
+    Committees.
 
     Creates the builtin user: admin.
     """
-    # Check whether the group pks 1 to 4 are free
-    if Group.objects.filter(pk__in=range(1, 5)).exists():
+    # Check whether the group pk's 1 to 5 are free.
+    if Group.objects.filter(pk__in=range(1, 6)).exists():
         # Do completely nothing if there are already some of our groups in the database.
         return
```

Comment:
Can you explain what this check should do? Why you filter from 1 to 6 and not from pk=1 to 5?